src/crimson/os/seastore/segment_manager/zns.cc

```diff
@@ -17,6 +18,29 @@ SET_SUBSYS(seastore_device);
 // limit the max padding buf size to 1MB
 #define MAX_PADDING_SIZE 1048576
 
+using z_op = crimson::os::seastore::segment_manager::zns::zone_op_t;
```

Comment:
just in case.. now that we are using C++20. we could use something like:

```c++
template <>
struct fmt::formatter<crimson::os::seastore::segment_manager::zns::zone_op_t> : fmt::formatter<std::string_view> {
  using enum crimson::os::seastore::segment_manager::zns::zone_op_t;
  std::string_view name = "Unknown";
  switch (s) {
  case ZONE_OPEN:
    name = "BLKOPENZONE";
    break;
  // ...
  }
};
```

Reply:
Initially I did like this, but then the lines felt un-necessarily long, so I used `using z_op = ...`
Will change it.

Reply:
no problem. just wanted to show that we could use `using enum` to expose the enum members. if you prefer a shorter name, probably we could use both of them, namely the type alias and `using enum`.
```c++
using zone_op = crimson::os::seastore::segment_manager::zns::zone_op_t;
/// ...
using enum zone_op;
```

Reply:
Yes, this feels better. Updated. Thanks